Formula/conftest.rb

```diff
@@ -0,0 +1,18 @@
+class Conftest < Formula
+  desc "Test your configuration files using Open Policy Agent"
+  homepage "https://www.conftest.dev/"
+  url "https://github.com/open-policy-agent/conftest/archive/v0.23.0.tar.gz"
+  sha256 "116bfd3c00fcaa31a3468cace6df5b03eeff0150d0d096aee53dd74c7c9647bd"
+  license "Apache-2.0"
+  head "https://github.com/open-policy-agent/conftest.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-X github.com/open-policy-agent/conftest/internal/commands.version=#{version}"
+  end
+
+  test do
+    assert_match "Test your configuration files using Open Policy Agent", shell_output("#{bin}/conftest --help")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
test for the `verify` command has been added